packages/context/README.md

```diff
@@ -160,7 +160,16 @@ Returns a function that can be used to navigate to another page. The returned fu
 
 #### `useApp()`
 
-Returns the currently loaded Evoke app.
+Returns an object with the following properties:
+
+-   `app`: The currently loaded Evoke app.
+-   `findDefaultPageSlugFor`: An asynchronous function that takes an `objectId` as a parameter and returns the default page slug for that object, if no default page slug is found and the object is a subtype, the page slug of the first ancestor with a default page will be used. It returns `undefined` if no default page slug is found for any ancestor type.
+
+Example usage:
+
+````javascript
```

Comment:
Is there some significance to having 4 backticks instead of 3?

Also, this code block is not terminated.